indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -1901,6 +2058,11 @@ protected boolean supportsPartitionExpiration()
     return false;
   }
 
+  public int getPartitionNumbers()
```

Comment:
nit:
```suggestion
  public int getPartitionsCount()
```

Reply:
Done. Thanks a lot for your review and approval!